.github/workflows/build-nvidia-images.yml

```
@@ -0,0 +1,51 @@
+name: Build nvidia docker images
+
+on:
+  push:
+    branches:
+      main
+    paths:
+      - conda/build_nvidia_image.sh
+      - .github/workflows/build-nvidia-images.yml
+  pull_request:
+    paths:
+      - conda/build_nvidia_image.sh
+      - .github/workflows/build-nvidia-images.yml
+
+# For setup-miniconda, see https://github.com/conda-incubator/setup-miniconda/issues/179
+defaults:
+  run:
+    shell: bash -x -e -l {0}
+env:
+  BUILD_ENVIRONMENT: build-linux-magma
+  DOCKER_REGISTRY: "docker.io"
+  DOCKER_BUILDKIT: 0
+  COMPOSE_DOCKER_CLI_BUILD: 0
+  DOCKER_ID: ${{ secrets.DOCKER_ID }}
+  DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+  IN_CI: 1
+  IS_GHA: 1
+  IMAGE_NAME: "nvcr.io/nvidia/cuda"
```

Maybe we should instead choose to upload these to `ghcr.io`
```suggestion
  IMAGE_NAME: "ghcr.io/pytorch/nvidia/cuda"
```

---

Done!